src/app/_components/Input.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+
+import { WarningCircle } from '@phosphor-icons/react/dist/ssr'
+
+import { Icon } from '@/components/Icon'
+
+import { capitalize } from '@/utils/capitalize'
+
+export type InputProps = {
+  label: string
+  error?: string
+} & React.ComponentProps<'input'>
+
+export function Input({ id, label, error, value = '', ...rest }: InputProps) {
+  return (
+    <div className="relative space-y-2">
+      <label className="text-sm font-bold text-brand-300" htmlFor={id}>
+        {capitalize(label)}
+      </label>
```

Comment:
The design for labels have changed a little bit since you started working on it:
https://www.figma.com/design/0pBfQs5tKI6bx5ypPIfVRA/FIL.org-V4-Prototypes?node-id=5285-76307&t=Kp5DZQTQ4NoPKmrn-4

Also it would be nice to encapsulate labels in their own `InputLabel` component, since we will use them with other inputs as well.

BUT, I realise that `NewsletterForm` does not use labels, so it can be done in a separate PR also :)